include/InstrumentTrack.h

```diff
@@ -229,6 +233,8 @@ class LMMS_EXPORT InstrumentTrack : public Track, public MidiEventProcessor
 	void newNote();
 	void endNote();
 
+public slots:
+	void processCCEvent(int controller);
```

Comment:
Picky, but I believe LMMS uses `( element )` format instead of `(element)`, so request you to change this for consistency

Reply:
Checking other code slices though `(element)` is used, so if this file uses `(element)` you can leave it as is

Reply:
We used to use `( foo )`, but `(foo)` is the standard now.
https://github.com/LMMS/lmms/wiki/Coding-Conventions#braces

Reply:
Yep, but most of the files are still on using `( element )`. I was suggesting to stick to that until a project-wide reformat takes place

Reply:
Since automatically reformatting is going to be a thing soon, when it comes to braces and small code format details I tend to use what is being used in the file I'm changing to keep it at least consistent (unless it's some terrible formatting mistake). On this file both styles are used, but the method right below mine uses `(element)`, so that's probably why I chose it. As @PhysSong pointed out, this is actually the standard now (not that I had that in mind when I wrote it), so should I keep it as it is?

Reply:
Yeah, I have no issues. I couldn't see the line below, so I assumed it uses the spaced brackets.

Reply:
Alright, thanks! I'll mark this as resolved then :)